### ISSUE TITLE
Zap with cbETH

### ISSUE THREAD

**Issue by micahalcorn** [OriginProtocol/origin-dollar#1659]
<img width="896" alt="Screenshot 2023-06-26 at 5 46 56 PM" src="https://github.com/OriginProtocol/origin-dollar/assets/273937/0d48527e-207a-4a7d-ad05-5dfce935be84">

**Comment by DanielVF** [comment]
We could just make the front end use curve to convert it.

**Comment by micahalcorn** [comment]
> We could just make the front end use curve to convert it.

True, but Curve liquidity for cbETH isn't particularly deep compared to the others. Whether we do this with a zap or the frontend (if at all), we may want to make it clear that cbETH is not a supported collateral type despite the convenience feature.

**Comment by sumpg41** [comment]
I like this idea - why not zap in with multiple non-supported LSTs as long as there's liquidity? cbETH, wBETH, ankrETH, ETHx, yETH...
